src/index.js

```diff
@@ -115,7 +138,11 @@ const getLikedPosts = (timestamp) => {
           for (let i = 0; i < likedPosts.length; i++) {
             const currentPost = likedPosts[i];
             if (Array.isArray(currentPost.photos)) {
-              currentPost.photos.forEach(pushToArray);
+              const filebase = cleanFileName(`${currentPost.blog_name}_${currentPost.timestamp}`);
+              currentPost.photos.forEach(pushToArray.bind(null, filebase));
+              if (includeNotes === true && currentPost.hasOwnProperty('caption')) {
+                saveNote(filebase, currentPost.caption);
```

Comment:
I didn't know that this could be an important thing to others. Great that you want to add this. 🙃